filesystem/squashfs_superblock.ksy

```diff
@@ -0,0 +1,112 @@
+meta:
+  id: squashfs_superblock
+  title: SquashFS superblock
+  file-extension:
+    - squashfs
+    - snap
+  endian: le
```

Comment:
```suggestion
  xref:
    justsolve: Squashfs
    wikidata: Q389314
  endian: le
```

Reply:
Why not rename the field to `fileformats`? `justsolve` doesn't sound like a descriptive name for that wiki.

Reply:
> Why not rename the field to `fileformats`? `justsolve` doesn't sound like a descriptive name for that wiki.

Well, that would be a question for another issue, but I don't think it's much beneficial. Right now, `justsolve` is an established identifier, so it doesn't make sense to change it arbitrarily.

In general, I don't care about the exact identifiers, it's just important to keep them consistent (so that no two different KSY specs use another names to refer to the same thing).

Reply:
This `justsolve` contains broken links, doesn't link to the real spec and assumes that `.sfs` extension without any references. I am not too enthusiastic to add it as a reference. I found more relevant results with Google than digging for the `justsolve` keyword and following it there.

I can't say that this `xref` to `justsolve` is human friendly, and if it is for robots, then wikidata is a better way to link various related resources, IMO.

Reply:
> assumes that `.sfs` extension without any references

Fair enough.